build.sh

```diff
@@ -121,14 +121,22 @@ fi
 
 ################################################################################
 # Configure, build, and install libnvstrings
-if buildAll || hasArg libnvstrings; then
+
+if buildAll || hasArg libnvstrings || hasArg libcudf; then
 
     mkdir -p ${LIBNVSTRINGS_BUILD_DIR}
```

Comment:
May want to rename consolidate the libnvstrings and libcudf build dir variables into a single one and use that instead. Kinda confusing the way it is.